SU2_CFD/src/solvers/CNEMONSSolver.cpp

```diff
@@ -1109,10 +1106,18 @@ void CNEMONSSolver::BC_Smoluchowski_Maxwell(CGeometry *geometry,
       Grad_PrimVar = nodes->GetGradient_Primitive(iPoint);
 
       /*--- Calculate specific gas constant --- */
+      //TODO: Move to fluidmodel?
       GasConstant = 0.0;
       for(iSpecies = 0; iSpecies<nSpecies; iSpecies++)
         GasConstant+=UNIVERSAL_GAS_CONSTANT*1000.0/Ms[iSpecies]*nodes->GetMassFraction(iPoint,iSpecies);
-
+      
+      /*--- Compute Gamma ---*/
+      //TODO: Move to fluidmodel?
```

Comment:
This is the lone (i think) instance where gamma is computed outside of fluidmodel.   Should be phased out.